interfaces/sourcegen/src/sourcegen/csharp/config.yaml

```diff
@@ -20,9 +20,10 @@ ignore_funcs:
 ret_type_crosswalk:
   const char*: string
   const double*: ReadOnlySpan<double>
-  const int*: ReadOnlySpan<int>
+  const int32_t*: ReadOnlySpan<int>
   char*: Span<byte>
   double*: Span<double>
+  int32_t: int
 
 # Parameter type crosswalks
```

Comment:
recommendation for clarity:

on line 19, change to
`# Return type crosswalks, maps C type names to C# type names where they differ`

on line 28, change to
`# Parameter type crosswalks, maps Span type used in interop call to appropriate return type`

Reply:
I ended up renaming the crosswalks, as the field names were a little misleading.